packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -142,6 +143,84 @@ class DateUtil {
     return undefined;
   }
 
+  /**
+   * Returns an object consisting of date input values based on the date format.
+   * @param {string} dateOrder - String containing the date input order. Possible values (MDY, DMY, YMD)
+   * @param {string} value - The date/placeholder string.
+   * @param {string} separator - The date separator.
+   */
+  static getDateInputValues(dateOrder, value, separator) {
+    if (!value) {
+      return false;
```

Comment:
Is it appropriate to return values or an object?

Reply:
Do you mean returning an object, something like { day: '', month: '', year: '' }, instead of returning false?

Reply:
This currently same return a boolean or an object which is inconsistent. Yes, this should return an empty object or { day: '', month: '', year: '' } depending on how the calling method can handle it.

Reply:
Addressed here: https://github.com/cerner/terra-framework/pull/1208/commits/01a9f2d9f673428c5a8dce65d64256d79c440f80